test/xstream.js

```diff
@@ -139,8 +139,8 @@ describe('Cyclic Router - XStream', () => {
         },
       }
 
-      const history = createServerHistory('/some/route')
-      const router = makeRouterDriver(history, switchPath)(xs.never(), XSAdapter)
+      const history = createMemoryHistory('/some/route')
+      const router = makeRouterDriver(history, switchPath)(xs.never())
```

Comment:
I think it works something like `createMemoryHistory({ initialPaths: [ '/some/route' ] })`